regression-tests.dnsdist/test_Spoofing.py

```diff
@@ -357,6 +359,50 @@ def testSpoofRawAction(self):
             self.assertEquals(expectedResponse, receivedResponse)
             self.assertEquals(receivedResponse.answer[0].ttl, 3600)
 
+    def testSpoofRawActionMulti(self):
+        """
+        Spoofing: Spoof a response from raw bytes
```

Comment:
```suggestion
        Spoofing: Spoof a response from several raw bytes
```
Just to prevent having two tests with the same description, so it's less confusing if that test fails :-)